bvm/ballerina-runtime/src/main/java/io/ballerina/runtime/api/types/ObjectType.java

```diff
@@ -28,6 +28,13 @@ public interface ObjectType extends StructureType, SelectivelyImmutableReference
 
     MethodType[] getMethods();
 
+    /**
+     * Get the type of the object init method.
```

Comment:
``` suggestion
     * Get the type of the user-defined object init method.
```

Reply:
Do we need to differentiate and specifically say the `init` as `user-defined init` in the API docs?

Reply:
Just checked the spec. It seems we don't specifically mention `user-defined init`. 
So, +1 to keep it as it is.